app/helpers/articles_helper.rb

```diff
@@ -56,4 +56,8 @@ def get_host_without_www(url)
   def utc_iso_timestamp(timestamp)
     timestamp&.utc&.iso8601
   end
+
+  def active_threads(**options)
```

Comment:
How do we feel about `active_threads` as the name of this method? Would it be better if it was named something else, like `sidebar_active_threads`, `active_thread_articles`, or something similar?

Reply:
I'm not the best at naming things but I like `active_threads`, the fact that they are now displayed in the sidebar is more related to presentation than anything else. Technically we could use the same function elsewhere 👀 

A possible alternative would be to use the terminology "active discussions" but it's definitely beyond the scope of this PR and I wouldn't recommend doing it before having a larger discussions about what an active discussion is :) 

+1 for your decision then!

Reply:
Sounds good to me! Thank you so much, @rhymes. 😄